.github/workflows/update-nightlies.yaml

```diff
@@ -116,6 +116,17 @@ jobs:
           fork: eventing-istio
           channel: eventing
           assignee: "@knative-extensions/eventing-wg-leads"
+        - nightly: eventing-gitlab
+          module: knative.dev/eventing
+          directory: ./third_party/eventing-latest
+          files: eventing-crds.yaml eventing-core.yaml eventing-tls-networking.yaml
```

Comment:
Not sure though if we need the TLS manifests (`eventing-tls-networking.yaml`)

Reply:
Keeping them as a precaution might need ssl to setup webhooks. Let me verify this on local.

Reply:
I had removed as of now If I'll need it I'll add later